apps/crn-server/src/entities/user.ts

```diff
@@ -168,11 +170,13 @@ const isOrcidWorkType = (data: string): data is OrcidWorkType =>
 export const parseUserToResponse = (user: UserDataObject): UserResponse => {
   const displayName = `${user.firstName} ${user.lastName}`;
   const onboarded = typeof user.onboarded === 'boolean' ? user.onboarded : true;
-  return {
+  const response = {
     ...user,
     displayName,
     onboarded,
   };
+  delete response.connections;
```

Comment:
i guess you could destruct connections from the `user` and ignore it instead of mutating the object this way?

Reply:
done